src/client/rest/RESTMethods.js

```diff
@@ -377,6 +377,12 @@ class RESTMethods {
     return this.rest.makeRequest('get', Constants.Endpoints.channelMessage(channel.id, messageID), true);
   }
 
+  putGuildMember(guild, user, options) {
+    return this.rest.makeRequest('put', Constants.Endpoints.guildMember(guild.id, user.id), true, options).then(data =>
```

Comment:
Are you sure about this? Because i followed the same code from other methods like `deleteGuild`, `updateCurrentUser`, `updateGuild`, etc...
https://github.com/MikeKovarik/discord.js/blob/571712b26ade77f6582c5b18f7891f3aa5defde0/src/client/rest/RESTMethods.js#L244